src/figuresClasses.ts

```diff
@@ -1,19 +1,91 @@
 export interface Figure {
-
+  shape: string;
+  color: string;
```

Comment:
Let's make type of shape and color `union type` because you know exactly what values it might be